cert-manager/eventfilter/eventfilter.go

```diff
@@ -0,0 +1,20 @@
+package eventfilter
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+)
+
```

Comment:
Used in the controller to make sure to only look at deployment of our operator in our namespace